### PR TITLE
Btwisted=big panda 121517

### PR DESCRIPTION
**Pull request by btwisted**
Ready for @claire review.

### CODE REVIEW

File: _posts/02_pro/03_integrations/2017-05-04-bigpanda.md [hunk 1]
```diff
@@ -20,40 +20,36 @@ After you set up the integration, you can view real-time alerts based on the res
...
-  <li>To generate the App key, enter a name for the integration.</li>
+  <li>To generate the App key, enter a name for the integration and click the <b>Generate App Key</b> button.</li>
 </ol>
 
-[![generate app key](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/58834937.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/58834937.png)
+[![generate app key](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/bigPanda_generateAppkey.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/bigPanda_generateAppkey.png)
+
+The generated App Key displays.
+
+[![results app key](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/bigPanda-appKey.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/bigPanda-appKey.png)
```
Comment (claireriley) [inline _posts/02_pro/03_integrations/2017-05-04-bigpanda.md]: Do we want to blur this? Someone might try to use it.

File: _posts/02_pro/03_integrations/2017-05-04-bigpanda.md [hunk 2]
```diff
@@ -20,40 +20,36 @@ After you set up the integration, you can view real-time alerts based on the res
...
 <ol start="4">
   <li>Save the generated App key and the API key for use later.
 </li>
 </ol>
 
-[![note keys](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/58835014.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/58835014.png)
-
 ### Configuring Postman monitors
 
 1. In the **[Integrations](https://app.getpostman.com/dashboard/integrations){:target="_blank"}** page, select BigPanda from a list of Postman's 3rd party Integrations for Postman Pro users.
```
Comment (claireriley) [inline _posts/02_pro/03_integrations/2017-05-04-bigpanda.md]: > 1. In the **[Integrations](https://app.getpostman.com/dashboard/integrations){:target="_blank"}** page, select BigPanda from a list of Postman's 3rd party Integrations for Postman Pro users.

option to tell the user where they should be in the beginning of the sentence (someone might read the first bit and think they're still supposed to be in BigPanda's dashboard) ->

> 1. In your Postman Dashboard, navigate to **[Integrations](https://app.getpostman.com/dashboard/integrations){:target="_blank"}** then select BigPanda from a list of Postman's 3rd party Integrations for Postman Pro users.

Comment (claireriley) [reply _posts/02_pro/03_integrations/2017-05-04-bigpanda.md]: Also you can't "select" this without clicking "View Details" in step 2..maybe

> 1. In your Postman Dashboard, navigate to Integrations{:target="_blank"} then _locate_ BigPanda in the list of Postman's 3rd party Integrations for Postman Pro users.

File: _posts/02_pro/03_integrations/2017-05-04-bigpanda.md [hunk 3]
```diff
@@ -20,40 +20,36 @@ After you set up the integration, you can view real-time alerts based on the res
...
 
-[![bigpanda integrations](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/integrations_bigPanda1.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/integrations_bigPanda1.png)
-
-
-<ol start="2">
-  <li>Click the <b>View Details</b> button to see information about BigPanda and how it can provide real-time alerting based on the results of your Postman monitors. 
-</li>
-</ol>
+2. Click the **View Details** button to see information about BigPanda and how it can provide real-time alerting based on the results of your Postman monitors.
```
Comment (claireriley) [inline _posts/02_pro/03_integrations/2017-05-04-bigpanda.md]: "alerting" sounds awkward - maybe just "alerts"?

**Comment by Claire** [comment]
@btwisted In case you haven't yet figured it out, I'm a different Claire than the one you wanted to ask to review this PR.

**Pull request closed without merging**